### ISSUE TITLE
The tabs region changes size when the app is moved to monitors with different DPI scaling

### ISSUE THREAD

**Issue by rbeesley** [microsoft/terminal#1336]
# Environment

```none
Windows build number: 18362.175
Windows Terminal version: 0.2.1703.0
```

# Steps to reproduce
1. Open Terminal on a monitor with 100% scaling.
2. Drag the Terminal app to a monitor with 150% scaling.

# Expected behavior
There should still be room on the title bar to click and move the app.

# Actual behavior
The gap between the drop down and the minimize button shrinks to nothing, maybe even with a little overlap. Because of #528, this means that it is likely you won't be able to move the app back after you place it. There is a small gap between the client area and below the system buttons, because the tabs are probably a little too tall for the region, but that is the only space that you can use to drag the app.

**Comment by zadjii-msft** [comment]
Could we get some screenshots? 

I think this will be solved by #929 _and it's follow-ups_, but I want to make sure.

**Comment by DHowett-MSFT** [comment]
I'm willing to bet that this was caused by the switch from `GetDpiForWindow` to `GetCurrentDpiScale` [here](https://github.com/microsoft/terminal/pull/829/files#diff-e8986dd365aa23a5fbc39a8f26e76d58L94)... ☹️

**Comment by DHowett-MSFT** [comment]
Wait nevermind ignore me

**Comment by rbeesley** [comment]
100% Scaling:
![image](https://user-images.githubusercontent.com/3703112/59810044-3dcd1900-92b8-11e9-8ede-65e1c43c453d.png)

150% Scaling:
![image](https://user-images.githubusercontent.com/3703112/59810070-59d0ba80-92b8-11e9-8506-9e79e02d05e5.png)

**Comment by DHowett-MSFT** [comment]
This will definitely be solved by #929.

**Comment by aminmesbahi** [comment]
Same issue when moving between Surfacebook and an FHD monitor
![issue](https://user-images.githubusercontent.com/866928/60536545-bbaf0c80-9d1b-11e9-9699-ae3105c1f1ff.PNG)
